File: Chapter9ZeusStudy1_3/pluginapi/src/main/java/com/mars/pluginapi/BaseActivity.java
package com.mars.pluginapi;

import android.app.Activity;
import android.content.res.Resources;

/**
 * Created by geyan on 2021/1/19
 */
public class BaseActivity extends Activity {

    @Override
    public Resources getResources() {
        return PluginManager.mCurResources;
    }
}

File: ZeusStudy1_1/pluginapi/src/main/java/com/mars/pluginapi/BaseActivity.kt
package com.mars.pluginapi

import android.content.res.Resources
import androidx.appcompat.app.AppCompatActivity

/**
 * Created by geyan on 2020/11/6
 */
open class BaseActivity: AppCompatActivity() {

    override fun getResources(): Resources {
        return PluginManager.getResources()
    }
}